kde-frameworks/kio/kio-5.109.0.ebuild

```diff
@@ -54,7 +52,11 @@ RDEPEND="
 		sys-apps/attr
 		virtual/acl
 	)
-	handbook? ( =kde-frameworks/kdoctools-${PVCUT}*:5 )
+	handbook? (
```

Comment:
I imagine these 3 are all BDEPEND.

Reply:
@thesamesam for reference: https://github.com/gentoo/gentoo/pull/32487/files#r1320738409